### ISSUE TITLE
[Dialog][Panel][Accessibility] The heading is always set at level 2.

### ISSUE THREAD

**Issue by fsteffi** [OfficeDev/office-ui-fabric-react#11666]
Environment Details:​​
Browser Versions:Chrome Version 78.0.3904.97 
Windows Version: Windows10​​​

Repro Steps:​​
1. Hit the application URL
2. Repeat the same steps as ( Select application as "Sales hub" from the list of application--> From the change area which is present in the bottom select "App Settings"-->Navigate to "Site Map" and then Navigate to "Goal Management" and select "Forecast Configuration")
3. Fast Track Your Forecast Screen will appears.
4. Click on "Org chart forecast"
5. Forecast Configuration screen will appears
6. Fill all the mandatory field and click on "next button".
7. Again click on next button.
8. Layout and columns screen will appears and then click on +Add from option set button and from the options select "Forecast category" and click on "Choose selected"
9. Auto Configure popup will appears
10.Launch accessibility insights for web and ad hoc tools and then select heading.

Actual:​
The heading for the pop-up is displayed as h2.

Same issue is observed in below screens:
1)Confirm delete popup
2)Delete the column.

Expected:​
The heading should be at level h1, to maintain the structure.

User Impact:​
Screen reader users will think there might be a hidden h1 heading that as not been announced , as well as the structure will not be defines well for the users.

![1640412_A11y_D365ForecastConfiguration_Layoutandcolumns_Auto-configurecolumns_A14W](https://user-images.githubusercontent.com/24580570/72140101-0047ee80-33b6-11ea-9e24-0c0b613bf954.png)

**Comment by xugao** [comment]
@fsteffi - your repro details are your application-specific. in general, pls make your repro simple and minimal 

also, i dont think the p2 is defined by fabric control. see: https://developer.microsoft.com/en-us/fabric#/controls/web/dialog. if you think otherwise, please provide an example in codepen.

**Comment by fsteffi** [comment]
@xugao
Reproduction of the issue in a codepen:
https://fabricweb.z5.web.core.windows.net/oufr/7.22.1/#/examples/dialog
and
https://developer.microsoft.com/en-us/fabric#/controls/web/dialog

Steps to repro:
Launch accessibility insights for web in chrome and select the heading toggle button.

![image](https://user-images.githubusercontent.com/24580570/72242492-9d519400-360f-11ea-8755-c52b48ac8d4c.png)

**Comment by xugao** [comment]
Agreed. The heading level should be customizable by user. will look into fixing
